packages/frontend/src/app/(sticky-header)/article/[slug]/article.tsx

```diff
@@ -68,14 +68,30 @@ const getPostContents = (post: any) => {
         }
       : undefined
   })
-  const orderedAuthors: Author[] = []
-  AUTHOR_ROLES_IN_ORDER.forEach((authorRole) => {
-    const authorsOfRole = authors?.filter(
-      (author: any) =>
-        author?.role?.split('、')?.includes(authorRole) && author?.link
-    )
-    orderedAuthors.push(...(authorsOfRole ?? []))
-  })
+
+  // Sort authors by AUTHOR_ROLES_IN_ORDER
+  const orderedAuthors = authors
+    ?.filter((author: any) => author?.link)
```

Comment:
Question:
Do we need to avoid using `any` in our coding convention?

Reply:
fixed.